<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Open PlayStation 2 Loader PC Tools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/application</normaloff>:/icons/application</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTreeWidget" name="mp_tree_games">
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="mp_widget_game_details" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="mp_layout_game_details">
            <item>
             <widget class="QLabel" name="mp_label_game_id">
              <property name="styleSheet">
               <string notr="true">QLabel {
	font-weight: bold;
}</string>
              </property>
              <property name="text">
               <string notr="true">Game ID</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="mp_label_media_type">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string notr="true">DVD</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="mp_label_installation_type">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string notr="true">UL</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="mp_groupbox_cover">
              <property name="title">
               <string>Cover</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="mp_label_cover">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #dfdfdf;</string>
                 </property>
                 <property name="text">
                  <string notr="true"/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="mp_toolbtn_change_cover">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                 <property name="icon">
                  <iconset theme="document-edit">
                   <normaloff>.</normaloff>.</iconset>
                 </property>
                 <property name="popupMode">
                  <enum>QToolButton::MenuButtonPopup</enum>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextBesideIcon</enum>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="mp_groupbox_icon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Icon</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="mp_label_icon">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>64</width>
                     <height>64</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>64</width>
                     <height>64</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: #dfdfdf;
</string>
                   </property>
                   <property name="text">
                    <string notr="true"/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QToolButton" name="mp_toolbtn_change_icon">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="sizeIncrement">
                  <size>
                   <width>140</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                 <property name="icon">
                  <iconset theme="document-edit">
                   <normaloff>.</normaloff>.</iconset>
                 </property>
                 <property name="popupMode">
                  <enum>QToolButton::MenuButtonPopup</enum>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextBesideIcon</enum>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mp_btn_manage_arts">
              <property name="text">
               <string>Manage Arts</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>168</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mp_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="mp_menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="mp_action_open_file"/>
    <addaction name="mp_action_reload_file"/>
    <addaction name="separator"/>
    <addaction name="mp_action_settings"/>
    <addaction name="separator"/>
    <addaction name="mp_action_quit"/>
   </widget>
   <widget class="QMenu" name="mp_menu_game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="mp_action_add_game"/>
    <addaction name="mp_action_to_iso"/>
    <addaction name="separator"/>
    <addaction name="mp_action_rename_game"/>
    <addaction name="mp_action_delete_game"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="mp_action_about_qt"/>
    <addaction name="mp_action_about"/>
   </widget>
   <addaction name="mp_menu_file"/>
   <addaction name="mp_menu_game"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="mp_statusbar"/>
  <widget class="QToolBar" name="mp_toolbar">
   <property name="windowTitle">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mp_action_open_file"/>
   <addaction name="mp_action_reload_file"/>
   <addaction name="separator"/>
   <addaction name="mp_action_add_game"/>
   <addaction name="mp_action_delete_game"/>
   <addaction name="mp_action_rename_game"/>
  </widget>
  <action name="mp_action_open_file">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/document-open</normaloff>:/icons/document-open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open OPL Catalog...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mp_action_add_game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/archive-insert</normaloff>:/icons/archive-insert</iconset>
   </property>
   <property name="text">
    <string>&amp;Install Game...</string>
   </property>
   <property name="toolTip">
    <string>Install Game</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="mp_action_delete_game">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/document-close</normaloff>:/icons/document-close</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Game</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="mp_action_rename_game">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/document-edit</normaloff>:/icons/document-edit</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename...</string>
   </property>
   <property name="toolTip">
    <string>Rename Game</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="mp_action_reload_file">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/view-refresh</normaloff>:/icons/view-refresh</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="mp_action_about">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="mp_action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="mp_action_about_qt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="mp_action_settings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="mp_action_to_iso">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>R&amp;ecover ISO Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mp_action_open_file</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadGameCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_rename_game</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>renameGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_reload_file</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>reloadUlConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_add_game</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_delete_game</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_about</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_about_qt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_settings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_action_to_iso</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>gameToIso()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_tree_games</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>gameSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_tree_games</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>renameGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_toolbtn_change_icon</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_toolbtn_change_cover</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setCover()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_btn_manage_arts</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manageArts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>965</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadGameCollection()</slot>
  <slot>gameSelectionChanged()</slot>
  <slot>renameGame()</slot>
  <slot>reloadUlConfig()</slot>
  <slot>addGame()</slot>
  <slot>deleteGame()</slot>
  <slot>about()</slot>
  <slot>aboutQt()</slot>
  <slot>setCover()</slot>
  <slot>removeCover()</slot>
  <slot>setIcon()</slot>
  <slot>removeIcon()</slot>
  <slot>showSettings()</slot>
  <slot>gameToIso()</slot>
  <slot>manageArts()</slot>
 </slots>
</ui>
